earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/__init__.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3
```

Comment:
These should all be python3.8, not python3

Reply:
done